ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/CommitterSpec.scala

```diff
@@ -192,7 +185,11 @@ class CommitterSpec
         }
       }
 
-      committer.run(None, aDamlSubmission, Ref.ParticipantId.assertFromString("test"), Map.empty)
+      committer.runWithPreExecution(
+        aDamlSubmission,
+        Ref.ParticipantId.assertFromString("test"),
+        Map.empty,
```

Comment:
Nit: named arguments(?)

Reply:
I agree but same argument as before, I'm mostly just moving code around so I would prefer not to have to touch it more than that